app/models/ext_management_system.rb

```diff
@@ -274,6 +274,17 @@ def connection_configurations=(options)
     options.each do |option|
       add_connection_configuration_by_role(option)
     end
+
+    drop_unused_connections(options)
```

Comment:
I think I like the name `delete_unused_connection_configurations` better here.

`drop_unused_connections` sounds like you're dropping active open connections.